pkg/application.go

```diff
@@ -0,0 +1,53 @@
+package pkg
```

Comment:
As these are the internals to power a demo app, can we make this package private?  Move it internal/pkg so that it cannot be imported outside of the modeule.